angel_system/berkeley/detectron2/data/datasets/kitware_cooking.py

```diff
@@ -0,0 +1,56 @@
2, 'name': 'paper filter (quarter)', 'instances_count': 102, 'def': '', 'synonyms': [], 'image_count': 102, 'frequency': '', 'synset': ''}, {'id': 23, 'name': 'paper filter (semi)', 'instances_count': 43, 'def': '', 'synonyms': [], 'image_count': 43, 'frequency': '', 'synset': ''}, {'id': 24, 'name': 'paper filter + filter cone', 'instances_count': 39, 'def': '', 'synonyms': [], 'image_count': 39, 'frequency': '', 'synset': ''}, {'id': 25, 'name': 'paper filter + filter cone + mug', 'instances_count': 251, 'def': '', 'synonyms': [], 'image_count': 251, 'frequency': '', 'synset': ''}, {'id': 26, 'name': 'paper filter bag', 'instances_count': 655, 'def': '', 'synonyms': [], 'image_count': 655, 'frequency': '', 'synset': ''}, {'id': 27, 'name': 'paper towel', 'instances_count': 412, 'def': '', 'synonyms': [], 'image_count': 412, 'frequency': '', 'synset': ''}, {'id': 28, 'name': 'scale (off)', 'instances_count': 688, 'def': '', 'synonyms': [], 'image_count': 688, 'frequency': '', 'synset': ''}, {'id': 29, 'name': 'scale (on)', 'instances_count': 80, 'def': '', 'synonyms': [], 'image_count': 80, 'frequency': '', 'synset': ''}, {'id': 30, 'name': 'switch', 'instances_count': 513, 'def': '', 'synonyms': [], 'image_count': 513, 'frequency': '', 'synset': ''}, {'id': 31, 'name': 'thermometer (close)', 'instances_count': 453, 'def': '', 'synonyms': [], 'image_count': 453, 'frequency': '', 'synset': ''}, {'id': 32, 'name': 'thermometer (open)', 'instances_count': 147, 'def': '', 'synonyms': [], 'image_count': 147, 'frequency': '', 'synset': ''}, {'id': 33, 'name': 'timer', 'instances_count': 624, 'def': '', 'synonyms': [], 'image_count': 624, 'frequency': '', 'synset': ''}, {'id': 34, 'name': 'timer (20)', 'instances_count': 29, 'def': '', 'synonyms': [], 'image_count': 29, 'frequency': '', 'synset': ''}, {'id': 35, 'name': 'timer (30)', 'instances_count': 28, 'def': '', 'synonyms': [], 'image_count': 28, 'frequency': '', 'synset': ''}, {'id': 36, 'name': 'timer (else)', 'instances_count': 111, 'def': '', 'synonyms': [], 'image_count': 111, 'frequency': '', 'synset': ''}, {'id': 37, 'name': 'trash can', 'instances_count': 29, 'def': '', 'synonyms': [], 'image_count': 29, 'frequency': '', 'synset': ''}, {'id': 38, 'name': 'used paper filter', 'instances_count': 60, 'def': '', 'synonyms': [], 'image_count': 60, 'frequency': '', 'synset': ''}, {'id': 39, 'name': 'used paper filter + filter cone', 'instances_count': 69, 'def': '', 'synonyms': [], 'image_count': 69, 'frequency': '', 'synset': ''}, {'id': 40, 'name': 'used paper filter + filter cone + mug', 'instances_count': 8, 'def': '', 'synonyms': [], 'image_count': 8, 'frequency': '', 'synset': ''}, {'id': 41, 'name': 'water', 'instances_count': 603, 'def': '', 'synonyms': [], 'image_count': 603, 'frequency': '', 'synset': ''}, {'id': 42, 'name': 'water + coffee grounds + paper filter + filter cone + mug', 'instances_count': 252, 'def': '', 'synonyms': [], 'image_count': 252, 'frequency': '', 'synset': ''}]
```

Comment:
Ouch this line... Should this be so hard coded vs. a config file? Or is this python file supposed to act as a config file itself?

Reply:
yeah, this is kinda trying to be a config file. My main goal was just to have this in one place instead of multiple files (how it currently is). I think there's work to set up a better solution (actual config files?) but I'm leaning towards that being post-demo